From 75c58eda090c0864d6583d213941913b20edf1dd Mon Sep 17 00:00:00 2001
From: bluesky <chenchongbiao@deepin.org>
Date: Thu, 3 Apr 2025 11:33:39 +0800
Subject: [PATCH] feat: Add loong64 support

add loong64 support to build rootfs for mmdebstrap

Log: add loong64 support
---
 mmdebstrap | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mmdebstrap b/mmdebstrap
index 0abdfc3..bff7d2f 100755
--- a/mmdebstrap
+++ b/mmdebstrap
@@ -5055,6 +5055,7 @@ sub main() {
             ppc64    => 'ppc64',
             ppc64el  => 'ppc64le',
             riscv64  => 'riscv64',
+            loong64  => 'loongarch64',
             s390x    => 's390x',
             sh4      => 'sh4',
             sparc    => 'sparc',
-- 
2.45.2

